lib/analytics/ga4_import/report_request_builder.rb

```diff
@@ -0,0 +1,59 @@
+module Analytics
+  module Ga4Import
+    class ReportRequestBuilder
+      attr_accessor :offset, :limit
+
+      START_DATE = 13
```

Comment:
This could use a better name reflecting what it is, and maybe use `#days` to be more descriptive. I'd also say maybe setting it to 14 and then taking the days off of today rather than yesterday to make it more obvious what is going on?

Maybe:

```ruby
REPORT_WINDOW = 14.days
```